EDITORS.md

```diff
@@ -185,6 +186,10 @@ Note that there might be overlapping functionality when using it with RubyMine,
 
 [Ruby LSP plugin](https://plugins.jetbrains.com/plugin/24413-ruby-lsp)
 
+## BBEdit
+
+See https://github.com/Shopify/ruby-lsp/issues/2365 for the latest information.
```

Comment:
I have the PR open to address the issue with BBEdit partially. Can we put configuration instructions here rather than pointing to the issue?